charts/k8ssandra/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 0.46.0
-appVersion: 3.11.7
+version: 0.47.0
+appVersion: 3.11.10
```

Comment:
What is the significance of changing this version? Guessing: it's just metadata, and that its intent is to say which version of C* we run by default?

Reply:
Yeah, sorry, this is unrelated, I just noticed that when we switched the default to .10 we didn't update this here.